test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatConfigTests.cs

```diff
@@ -0,0 +1,109 @@
+using System.Linq;
+using NuGet.CommandLine.XPlat;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.XPlat.FuncTest
+{
+    [Collection("NuGet XPlat Config Test Collection")]
+    public class XPlatConfigTests
+    {
+
+        [Fact]
+        public void ConfigPathsCommand_ListConfigPathsWithArgs_Success()
+        {
+            // Arrange
+            using (var testInfo = new TestInfo("NuGet.Config"))
+            {
+                var args = new[]
+                {
+                    "config",
+                    "paths",
+                    testInfo.WorkingPath
+                };
+                var log = new TestCommandOutputLogger();
+
+                // Act
+                var exitCode = Program.MainInternal(args.ToArray(), log);
+
+                // Assert
+                Assert.Equal(string.Empty, log.ShowErrors());
+                Assert.Equal(0, exitCode);
+                Assert.Contains(Path.Combine(testInfo.WorkingPath.Path, "NuGet.Config"), log.Messages);
+            }
+        }
+
+        [Fact]
+        public void ConfigPathsCommand_ListConfigPathsNonExistingDirectory_Fail()
+        {
+            // Arrange
+            using (var testInfo = new TestInfo("NuGet.Config"))
+            {
+                var args = new[]
+                {
+                    "config",
+                    "paths",
+                    @"C:\Test\NonExistingRepos"
+                };
+                var log = new TestCommandOutputLogger();
+
+                // Act
+                var exitCode = Program.MainInternal(args.ToArray(), log);
+                var expectedError = @"The path 'C:\Test\NonExistingRepos' doesn't exist.";
+
+                // Assert
+                Assert.Contains(expectedError, log.ShowErrors());
+                Assert.Equal(1, exitCode);
+            }
+        }
+
+        internal class TestInfo : IDisposable
+        {
+            public static void CreateFile(string directory, string fileName, string fileContent)
+            {
+                if (!Directory.Exists(directory))
+                {
+                    Directory.CreateDirectory(directory);
+                }
+
+                var fileFullName = Path.Combine(directory, fileName);
+                CreateFile(fileFullName, fileContent);
+            }
+
+            public static void CreateFile(string fileFullName, string fileContent)
+            {
+                using (var writer = new StreamWriter(fileFullName))
+                {
+                    writer.Write(fileContent);
+                }
+            }
+
+            public TestInfo(string configPath)
+            {
+                WorkingPath = TestDirectory.Create();
+                ConfigFile = configPath;
+                CreateFile(WorkingPath.Path,
+                           Path.GetFileName(ConfigFile),
+                           $@"
+<configuration>
+    <packageSources>
+        <add key=""Foo"" value=""https://contoso.com/v3/index.json"" />
```

Comment:
```suggestion
        <add key=""Foo"" value=""https://contoso.test/v3/index.json"" />
```

`.test` is a reserved top level domain, so we can be confident that this domain won't ever exist. Using `.test` domains for fake examples makes sure that if a mistake is made, we're not sending HTTP requests to a potentially real website.